setup-homebrew/action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
     description: Token to be used for GitHub authentication (if using private repositories). This token will persist through other steps for the duration of the job.
     required: false
     default: ""
+  brew-gh-api-token:
```

Comment:
NB: The `token` setting above seems to be used exclusively for private repo access, so I added this as a separate setting to avoid widening the former's scope. It also has a default unlike `token`.